test/crypto/ecdh.js

```diff
@@ -281,3 +283,77 @@ describe('ECDH key exchange @lightweight', function () {
     });
   });
 });
+
+describe('KDF parameters', function () {
```

Comment:
Maybe add that this is testing the forwarding `kdf params`, and if possible, test also the non-forwarding case.